src/deadline/job_attachments/README.md

```diff
@@ -41,6 +41,8 @@ These snapshots are encapsulated in one or more [`asset_manifests`](asset_manife
 
 When starting work, the worker downloads the manifest associated with your job, and recreates the file structure of your submission locally, either downloading all files at once, or as needed if using the [virtual][vfs] job attachments filesystem type. When a task completes, the worker creates a new manifest for any outputs that were specified in the job submission, and uploads the manifest and the outputs back to your S3 bucket.
 
+Manifest files are written to a `manifests` directory within each job bundle that is added to the job history (default: `~/.deadline/job_history`) if submitted through the GUI, or written to a temporary directory if submitted through the CLI with the `--save-manifests` flag. The file path inside the `manifests` directory corresponds to the S3 manifest path in the submitted job's job attachments metadata.
```

Comment:
This wouldn't be accurate any more since it's now either the temp dir or job bundle, right? 
`default: ~/.deadline/job_history`

Also whats the reasoning for why we would want to save in a temp directory if submitted with the CLI, vs the job bundle if submitted through GUI? I would think we would want these actions to be equivalent, just with a different interface.

Reply:
When you submit a job through the GUI, we copy the job bundle to the job history directory, which is in `~/.deadline/job_history` by default but can be changed. We don't do this for CLI submissions currently. So, for GUI submissions, we write the manifests to the copied job bundle history directory. As Mark suggested above, for CLI submissions we just write the manifests to a temporary directory since it doesn't make sense to have a job bundle history directory that just contains manifests.

Reply:
I see, thanks for the explanation